LanguageFeatures/Records/record_type_literals_A01_t01.dart

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion There is no record type literal syntax that can be used as an
+/// expression, since it would be ambiguous with other existing syntax:
+///
+/// var t = (int, String);
+/// This is a record expression containing two type literals, [int] and
+/// [String], not a type literal for a record type.
+///
+/// @description Checks that there is no record type literal syntax that can be
+/// used as an expression
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=records
+
+import "../../Utils/expect.dart";
+
+main() {
+  var t1 = (int, String);
+  var t2 = (int i, String s);
```

Comment:
I think this would be a syntax error, because `int i` is not an expression, and it is not of the form `id: e` where `id` is an identifier and `e` is an expression.